common/src/main/scala/skinny/ParamType.scala

```diff
@@ -79,22 +79,24 @@ object ParamType {
     case v: Short => scala.math.BigDecimal(v)
   })
   case object String extends AbstractParamType({
+    case v: String if v.isEmpty => null // don't trim
```

Comment:
I see. Indeed, we should care about such a case. It's not good for existing apps to change this behavior now.

@namutaka How about defining your own `ParamType` in your case?

```scala
case object NonEmptyString extends AbstractParamType({
  case v: String if v.isEmpty => null // don't trim
  ..
})
```

If lots of users need the above case object, we may need to consider put it into framework.